components/library/objfw/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2023, Jonathan Schleifer. All rights reserved.
+#
+
+BUILD_BITS = 32_and_64
+USE_PARALLEL_BUILD = yes
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME = objfw
+COMPONENT_VERSION = 1.0.3
+COMPONENT_SUMMARY = A portable, lightweight framework for the Objective-C language
+COMPONENT_SRC = $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE = $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH = sha256:1c81d7d03578b2d9084fc5d8722d4eaa4bdc2f3f09ce41231e7ceab8212fae17
+COMPONENT_ARCHIVE_URL = https://objfw.nil.im/downloads/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL = https://objfw.nil.im/
+COMPONENT_FMRI = library/objfw
+COMPONENT_CLASSIFICATION = Development/Objective C
+COMPONENT_LICENSE = QPL OR GPLv3 OR GPLv2
```

Comment:
I suggest to copy the relevant part of `README.md` related to licensing to `objfw.license`.  Then you could add the file to manifests as 4th `license` action (see the `sample-manifest.p5m`).

Reply:
This is in direct conflict with the changes @Toasterson requested above. This is what I initially had.

Reply:
@Toasterson [suggested](https://github.com/OpenIndiana/oi-userland/pull/13946#pullrequestreview-1651745644) to add three files, not to remove `objfw.license`.

Reply:
Ah, I see. Done!